src/EventHandler.h

```diff
@@ -22,42 +23,75 @@ class EventHandler
 public:
 	explicit EventHandler(std::string name);
 
-	const char* Name() const { return name.data(); }
+	const char* Name() const
```

Comment:
Is there a chance you're using a more recent clang-format version than the one in `.pre-commit-config.yaml` (13.0.0) or you didn't run `pre-commit run` before/after the commit?

Running `pre-commit run -a` within Zeek reverts most of the spurious newline changes. Unfortunately it doesn't seem Zeek's CI setup currently runs the pre-commit check, but I haven't understood why.

Reply:
Cirrus doesn't, but there should be a Github Action that runs it. It doesn't appear to have run on this PR though.

Reply:
It ran previously, actually: https://github.com/zeek/zeek/pull/2929/checks?check_run_id=12577202183

Not sure what's going on. Maybe on a re-push it'll :crossed_fingers:

Reply:
```
$ clang-format --version
clang-format version 16.0.1
```
I switched to using `pre-commit` :)